gnosis/safe-cmm-app-react
# Total value claimable

# :warning: Merge after https://github.com/gnosis/safe-cmm-app-react/pull/195 :warning: 

# Description

Adding `Total value claimable` to closed tab.

On load:
![screenshot_2020-11-30_14-46-48](https://user-images.githubusercontent.com/43217/100675255-78473c00-331b-11eb-9dc1-16709c0ea37a.png)

Loaded:
![screenshot_2020-11-30_14-47-02](https://user-images.githubusercontent.com/43217/100675262-7bdac300-331b-11eb-8a51-d669a651ec89.png)

# To Test
1. Make sure you have at least 1 closed strategy.
1. Go to strategies page
1. Go to closed tab

- [ ] The strategy should show a loader in the `Total value claimable` column, while loading
- [ ] The strategy should show an amount in each token the strategy holds in the `Total value claimable` column, after loaded

# Background

N/A
